bot/exts/backend/sync/__init__.py

```diff
@@ -1,7 +1,7 @@
 from bot.bot import Bot
-from .cog import Sync
 
 
 def setup(bot: Bot) -> None:
     """Load the Sync cog."""
+    from ._cog import Sync
```

Comment:
Might be worth adding a comment to explain why this import is now here. It's not immediately intuitive.

Reply:
Addressed by a66df29.